cartridges/int_cloudinary/cartridge/templates/default/include/cloudinaryShrinkwrap.isml

```diff
@@ -0,0 +1,18 @@
+<isset name="sitePref" value="${dw.system.Site.current.preferences.custom}" scope="page" />
+<isscript>
+    pdict.CLDCoreShrinkwrapJSURL = sitePref.CLDCoreShrinkwrapJSURL;
+</isscript>
+
+<script>
+    document.addEventListener('DOMContentLoaded', function() {
+        var shrinkwrapUrl = JSON.parse('<isprint value="${JSON.stringify(pdict.CLDCoreShrinkwrapJSURL)}" encoding="jsonvalue"/>');
+        if (document.querySelector('script[src="'+shrinkwrapUrl+'"]') === null) {
```

Comment:
Let's not wait for the DOM to load completely. Inject the script immediately and call `window.makeCloudinaryImagesResponsive();` once when the script is loaded and a second time when the DOM is loaded

Reply:
If we render the shrink wrap without waiting the DOMContent it throws error and doesn't render the video. I have attached the screenshot for reference.
![image (25)](https://github.com/user-attachments/assets/68c1077f-06dc-4d0b-b7f5-562005f32480)

Reply:
So this is a bug with the implementation of the Video Player. This code is about responsive images. We should support each one independently.

Please apply the responsive images as requested, and then check what is the issue with the Video Player

Reply:
There is a conflict between
https://unpkg.com/cloudinary-core@2.13.0/cloudinary-core-shrinkwrap.js


and
https://unpkg.com/cloudinary-video-player@1.10.1/dist/cld-video-player.min.js

both of the above js have cloudinary.Cloudinary.new()
so, in shrinkwrap there is no videoPlayer() method available, that's the reason it returns `cloudinary.Cloudinary.new(...).videoPlayer is not a funtion`.

Reply:
In case a page has both JS-responsive and the VP, we should load only the VP widget. The VP widget has both functionalities, so it would be more performant to not load the Core widget in this case

Reply:
true

Reply:
In responsive implementation we are using `cloudinary.default.Cloudinary.new()` we have removed default to make it similar with VP, but we don't have `responsive()` in `coudinary.Cloudinary.new()` in VP script. So we need to use both scripts.
![image (26)](https://github.com/user-attachments/assets/8d976d7d-0edd-4150-b26c-0eb0dd4e5a26)

Reply:
thanks @asad-rafter , that's make sense

Reply:
let me come back with a solution, as waiting for the DOM to load has its performance toll

Reply:
@asad-rafter The root cause for the conflict is that loading the core library (shrinkwrap) overrides the entry point to the Video Player.
Possible solutions:
1. Storing the access to the videoplayer on a global variable: `window.clVideoPlayer = cloudinary.Cloudinary;`
2. Making sure that the shrinkwrap is loaded before the videoplayer and then both functions (Responsive and new VideoPlayer) are available
 
 It seems that the first solution is easier to implement, although I don't have a strong preference here